Sources/AccioKit/Services/CarthageBuilderService.swift

```diff
@@ -16,7 +16,8 @@ final class CarthageBuilderService {
         framework: Framework,
         platform: Platform,
         swiftVersion: String,
-        alreadyBuiltFrameworkProducts: [FrameworkProduct]
+        alreadyBuiltFrameworkProducts: [FrameworkProduct],
+        toolchain: String? = nil
```

Comment:
Again, why does this default to `nil` here?

Reply:
It is nil to fallback to the default behaviour as before to avoid breaking changes